<launch>

  <node pkg="hector_motion_detection" type="motion_detection" name="motion_detection" output="screen" respawn="true">
    <remap from="image_percept" to="worldmodel/image_percept" />
    <param name="image_transport" value="compressed" />
  </node>

  <node pkg="libuvc_camera" type="camera_node" name="camera_node" output="screen" respawn="true">
    <remap from="image_raw" to="/arm_rgbd_cam/rgb/image_raw" />
  </node>

<rosparam file="$(find hector_motion_detection)/cfg/motion_detection_gazebo.yaml" />

</launch>
